tests/console/postgresql94.pm

```diff
@@ -11,17 +11,18 @@
 use base "consoletest";
 use strict;
 use testapi;
+use utils;
 
 sub run() {
     my $self = shift;
 
     select_console 'root-console';
 
     # install the postgresql94 client package
-    assert_script_run "zypper -n in postgresql94", 200;
+    zypper_call "in postgresql94";
 
     # check the postgresql94 client
-    assert_script_run "/usr/bin/psql --help", 200;
+    assert_script_run "/usr/bin/psql --help";
```

Comment:
I am not sure you should reduce the timeout here. The default is 90 seconds. I guess there is a reason why there is "200". Startup could take some time, I guess.